### ISSUE TITLE
Sponsor Logos_ "Texas Commission on the Arts Logo" & "ATX Music Logo"_Inconsistency in alt text.

### ISSUE THREAD

**Issue by HerinHentry** [mr-bobz/talarts#82]
Every other Sponsor logo image does not have logo in the alt text.
The mentioned 2 to be changed to "Texas Commission on the Arts" for consistency

**Comment by ruchitamanuja** [comment]
I have removed the logo from Alt text

**Comment by HerinHentry** [comment]
Excellent! Issue can be closed.
